api/app/Models/TestEntity.php

```diff
@@ -39,6 +39,16 @@ class TestEntity extends BaseModel {
     ];
 
     /**
+     * Get the access route for the entity.
+     *
+     * @return string
+     */
+    public function entityRoute()
+    {
+        return '/test/entities';
+    }
+
+    /**
```

Comment:
Can you remove this function entirely as we should be using the model factories in the seeders instead. This will fix the code coverage for this file.